test/integration.spec.js

```diff
@@ -0,0 +1,198 @@
+const isNodeRuntime = typeof window === "undefined"
+const expect = isNodeRuntime ? require("chai").expect : window.expect
+const convertToDataUrl = isNodeRuntime ? require("base64-arraybuffer").encode : x => x
+const MapdCon = isNodeRuntime ? require("../dist/node-connector.js").default : window.MapdCon
+const TPixel = isNodeRuntime ? require("../build/thrift/node/mapd_types.js").TPixel : window.TPixel
```

Comment:
This spec runs twice—once via mocha in the node environment, and once via karma in the browser environment. That way if someone writes new code or tests, we won't silently break one environment.